partition/roles/metal-core/README.md

```diff
@@ -0,0 +1,48 @@
+# metal-core
+Deploys metal-core container
+
+## Variables
+
+This role uses variables from [partition-defaults](/partition). So, make sure you define them adequately as well.
+
+You can look up all the default values of this role [here](defaults/main/main.yaml).
+
+| Name                            | Mandatory | Description                         |
+| ------------------------------- | --------- | ----------------------------------- |
+| metal_core_image_name           |           | Image name of the metal-core        |
+| metal_core_image_tag            | yes       | Image tag of the pixiecore          |
+| metal_core_port                 |           |                                     |
+| metal_core_cidr                 |           |                                     |
+| metal_core_log_level            |           |                                     |
+| metal_core_hmac_key             |           |                                     |
+| metal_core_change_boot_order    |           |                                     |
+| reconfigure_switch              |           |                                     |
```

Comment:
we should prefix all variables that are used in the metal-core role with "metal_core"

Reply:
Done